### ISSUE TITLE
Regarding the hyperparameter issue.

### ISSUE THREAD

**Issue by QinSY123** [LabShuHangGU/CCA#5]
Hello, I noticed that the code includes hyperparameters lambda and beta. If I want to adjust the bitrate, which parameter should I modify? According to the original paper, it should be lambda, but I am not entirely sure.

**Comment by MichTheEncoder** [comment]
Sorry for the confusion. In the code it should be `beta` which corresponds to `lambda` in the paper.

**Comment by QinSY123** [comment]
> Sorry for the confusion. In the code it should be `beta` which corresponds to `lambda` in the paper.

Thank you for your reply. I have two more questions. First, if beta controls the bitrate, why does it change during training, such as in line 480 of train.py? Second, I noticed that in distributed training, the learning rate (lr) is not multiplied by the number of GPUs. Could this affect the training? Also, did you use multi-GPU or single-GPU training?

**Comment by MichTheEncoder** [comment]
For the first question please refer to https://github.com/LabShuHangGU/CCA?tab=readme-ov-file#pretrained-models, which is a commonly used training method in LIC. For the second question we use a single graphics card for our pretrained model.
